graph/db/graph_test.go

```diff
@@ -4063,3 +4063,56 @@ func TestClosedScid(t *testing.T) {
 	require.Nil(t, err)
 	require.True(t, exists)
 }
+
+// testNodeAnn is a serialized node announcement message which contains an
+// address type (6) that LND is not aware of.
+var testNodeAnn = "01012674c2e7ef68c73a086b7de2603f4ef1567358df84bb4edaa06c" +
+	"f2132965b14e2434faab04170f0089216accbd79188fa3d40dbb0438bd89782cae" +
+	"27cc656bf60007800088082a69a2625e7a2a024b9a1fa8e006f1e3937f65f66c40" +
+	"8e6da8e1ca728ea43222a7381df1cc449605024b9a424c554549524f4e2d76302e" +
+	"31312e307263332d362d67663963613934650000001d0180c7caa8260702240061" +
+	"80000000d0000000005cd2a001260706204c"
```

Comment:
In the comment, I believe it should be `address type (5)` instead of `(6)`.

https://github.com/lightning/bolts/blob/5f31faa0b6e2cdbe32171d79464305f90bda9585/07-routing-gossip.md?plain=1#L295-L298